admin/acceptance.py

```diff
@@ -222,7 +232,7 @@ class RunOptions(Options):
          'One of fedora-20.'],
```

Comment:
Separate issue, but this is confusing - it has to be `fedora-20` and so the parameter is not currently useful, or this description is wrong? I'll create an issue for either if necessary.